src/__tests__/directives/long.test.js

```diff
@@ -48,10 +48,10 @@ import thrift2flow$Long from \\"long\\";
 export type MyStruct = {|
   posNum1?: ?thrift2flow$Long,
   posNum2?: ?thrift2flow$Long,
-  posNum3?: ?Buffer,
```

Comment:
To document, after chatting with Giancarlo I believe the main concern around this pull request was this line. It seems like this might break expected behavior because Int64 does't work in the browser.

Reply:
My understanding is these are decoded as Numbers in all cases in thriftrw. I've also added a test to verify this behavior.

Reply:
I wonder if using this would be more appropriate? https://tc39.github.io/proposal-bigint/

Reply:
Probably but then that would be a breaking change to thriftrw so I see it as orthogonal to thrift2flow which should just match whatever we have in thriftrw